enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -654,6 +687,118 @@ return (
 );
 ```
 
+#### _Interactive Dashboard_
+
+###### `dashcardMenu`
+
+This plugin allows you to add, remove, and modify the custom actions on the overflow menu of dashboard cards. The plugin
+appears as a dropdown menu on the top right corner of the card.
+
+The plugin's default configuration looks like this:
+
+```jsx
+const plugins = {
+  dashboard: {
+    dashcardMenu: {
+         withDownloads: true,
+         withEditLink: true,
+         customItems: [],
+    }
+  } 
+};
+```
+
+and can be used in the InteractiveDashboard like this:
+
+```jsx
+<InteractiveDashboard 
+    questionId={...}
+    plugins={{
+        dashboard: {
+            dashcardMenu: [your config here]
+        }
+    }}
+>
+```
+
+Take a look below to see how you can customize the plugin:
+
+###### Enabling/disabling default actions
+
+To remove the download button from the dashcard menu, set `withDownloads` to `false`. To remove the edit link from the
+dashcard menu, set `withEditLink` to `false`.
+
+```jsx
+const plugins = {
+  dashboard: {
+    dashcardMenu: {
+         withDownloads: false,
+         withEditLink: false,
+         customItems: [],
+    }
+  } 
+};
+```
+
+###### Adding custom actions to the existing menu:
+
+You can add custom actions to the dashcard menu by adding an object to the `customItems` array. Each element can either
+be an object or a function that takes in the dashcard's question, and outputs a list of custom items in the form of:
+
+```jsx
+{
+  iconName: string;
+  label: string;
+  onClick: () => void;
+  disabled?: boolean;
```

Comment:
Just curious - is the `disabled` prop supposed to be used dynamically, in practice? I suppose the plugins config needs to be constructed dynamically to use this then, otherwise the `disabled` state would always remain the same.